packages/react-select/src/Select.js

```diff
@@ -1917,14 +1786,27 @@ export default class Select extends Component<Props, State> {
   }
 
   renderLiveRegion() {
-    if (!this.state.isFocused) return null;
+    const { commonProps } = this;
+    const {
+      ariaSelection,
+      focusedOption,
+      focusedValue,
+      isFocused,
+      selectValue,
+    } = this.state;
+
+    const focusableOptions = this.getFocusableOptions();
+
     return (
-      <A11yText aria-live="polite">
-        <span id="aria-selection-event">
-          &nbsp;{this.state.ariaLiveSelection}
-        </span>
-        <span id="aria-context">&nbsp;{this.constructAriaLiveMessage()}</span>
-      </A11yText>
+      <LiveRegion
```

Comment:
Low effort improvement would be to include this in `getComponents`:

In Select.js
```tsx
const { LiveRegion } = this.getComponents()
```

And then for components/index.js:
```tsx
LiveRegion: ComponentType<LiveRegionProps>,
```

Reply:
I think exposing it in the public facing components is a near future goal but not for this release.